### DISCUSSION TITLE
Optimizations for old hardwares

### DISCUSSION THREAD

**Discussion by xuanhaivptthn** [ppy/osu#20310] [Q&A] [answered]
To all players using old/potato computers to play osu!; how is the performance between stable and lazer?
I tried to play beatmaps from the "triangles Beatmapping Contest" but it was way too laggy, hitsounds go off the rails after a while, sometime after I Alt+Tab to another windows a few time, lazer would just crash...

these are logs right after said crash, couldnt find any kind of useful info here...
[network.log](https://github.com/ppy/osu/files/9568274/network.log)
[performance.log](https://github.com/ppy/osu/files/9568275/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9568276/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9568278/updater.log)
[database.log](https://github.com/ppy/osu/files/9568279/database.log)

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Please be patient and wait for future optimisations, or upgrade your device. We'll eventually make it as fast (or faster) than stable on all devices we can support.
